zulipterminal/ui_tools/buttons.py

```diff
@@ -59,9 +61,10 @@ def update_count(self, count: int, text_color: Optional[str]=None) -> None:
             count_text = ''
         else:
             count_text = str(count)
-        self.update_widget(('unread_count', count_text), new_color)
 
-    def update_widget(self, count_text: Tuple[str, str],
+        self.update_widget((self.count_style, count_text), new_color)
+
+    def update_widget(self, count_text: Tuple[Optional[str], str],
```

Comment:
No I believe this is the right commit. This commit adds `count_style: Optional[str]=None,` to the `TopButton.__init__` function in line 22. Prior to this commit update_widget was called with `unread_count` therefore the type was `Tuple[str, str]` but due to the fact that in this commit `count_style` is now passed in instead of `unread_count` the type had to be changed